pkg/api/types.go

```diff
@@ -137,6 +137,9 @@ type OrchestratorProfile struct {
 	OrchestratorRelease string            `json:"orchestratorRelease"`
 	OrchestratorVersion string            `json:"orchestratorVersion"`
 	KubernetesConfig    *KubernetesConfig `json:"kubernetesConfig,omitempty"`
+	Registry            string            `json:"registry,omitempty"`
```

Comment:
Let's move these properties under `DcosConfig`. Documentation, example, and all code usage patterns will need to be adjusted accordingly. Thanks!